docs/setup/advanced/prefix-set.md

```diff
@@ -21,29 +27,37 @@ You can apply Prefix Sets to the following locations under the Management menu:
 
 + Rate Card :material-menu-right: Customer Rate Cards :material-menu-right: Profits :material-menu-right: :material-plus: :material-menu-right: Config :material-menu-right: [Prefix Set](https://docs.connexcs.com/customer-ratecard/#main-tab)
 
++ Customer :material-menu-right: Routing :material-menu-right: Ingress Routing :material-menu-right: Strategy :material-menu-right: [Prefix Set](https://docs.connexcs.com/customer-ratecard/#main-tab)
+
 ## Flags
-Use Flags to create a *Global CLI Blacklist* or a *Global Dialled Number Blacklist* (Aka Block List). This prevents all calls going through the system which match the defined prefix set. 
 
-!!! warning "Blacklist or Do Not Call list"
+Use Flags to create a *Global CLI Deny list* or a *Global Dialled Number Deny list* (Aka Block List). This prevents all calls going through the system which match the defined prefix set.
+
+!!! warning "Deny list or Do Not Call List"
     Use Flags for large amounts of numbers that change infrequently. If you want to manage a DNC list, please review our [**Do Not Call (DNC)**](https://docs.connexcs.com/dnc/) setup.
-    
- 
-## Example Dial Strings for Prefix sets
-Prefix Sets and all Dial String fields accept plain text or regular expressions. List only one prefix per line. Here are 3 samples, using plain-text, regex, or a mix:
 
-**Plan-Text**
-```
+## Example Dial Strings for Prefix Sets
+
+Prefix Sets and all Dial String fields accept plain text or regular expressions. List only one prefix per line.
+
+Here are 3 samples using plain-text, regex, or a mix:
+
+**Plain-Text**
+
+```bash
 441
 442
 ```
 
 **Regular Expression (RegEx)**
-```
+
+```bash

```

Comment:
should be text

Reply:
done
https://bani--connexcs-docs.netlify.app/setup/advanced/prefix-set/